tests/runtime/kubernetes/dump_kubernetes_logs.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What would you say to move this file under `tests/test_utils/runtime`?

Reply:
We can move it there, no problem.